newrelic/data_source_newrelic_synthetics_monitor_location.go

```diff
@@ -0,0 +1,77 @@
+package newrelic
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/newrelic/newrelic-client-go/pkg/synthetics"
+)
+
+func dataSourceNewRelicSyntheticsMonitorLocation() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNewRelicSyntheticsMonitorLocationRead,
+
+		Schema: map[string]*schema.Schema{
+			"label": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The label of the Synthetics monitor location.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The name of the Synthetics monitor location.",
+			},
+			"high_security_mode": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "The high security mode for the Synthetics monitor location",
```

Comment:
Can we go with the following here:

"Represents if high security mode is enabled for the location.  A value of true means that high security mode is enabled, and a value of false means it is disabled."